test/testUtils/utils.ts

```diff
@@ -28,7 +28,6 @@ export function getStartedApp(serverFolder: string, serviceName: string, fileExp
     let app = spawn(`cd ${serverFolder} && npm`, ["run", `start:${serviceName}:injected`], {
         env: {
             ...process.env, ...{
-                LUMIGO_TRACER_TOKEN: 't_123321',
```

Comment:
we remove it since the tests are writing to a file and we don't check the transaction in Lumigo